numpy/_core/src/multiarray/convert_datatype.cpp

```diff
@@ -3385,7 +3379,8 @@ object_to_object_get_loop(
         NpyAuxData **out_transferdata,
         NPY_ARRAYMETHOD_FLAGS *flags)
 {
-    *flags = NPY_METH_REQUIRES_PYAPI | NPY_METH_NO_FLOATINGPOINT_ERRORS;
+    *flags = (NPY_ARRAYMETHOD_FLAGS)(
```

Comment:
This is so weird: one can not even or flags in an enum together and not keep the same type?